src/main/java/org/openrewrite/kotlin/Assertions.java

```diff
@@ -146,27 +157,89 @@ public static SourceSpecs srcTestKotlin(SourceSpecs... kotlinSources) {
     }
 
     private static void acceptSpec(Consumer<SourceSpec<K.CompilationUnit>> spec, SourceSpec<K.CompilationUnit> kotlin) {
-        Consumer<K.CompilationUnit> userSuppliedAfterRecipe = kotlin.getAfterRecipe();
-        kotlin.afterRecipe(userSuppliedAfterRecipe::accept);
-        isFullyParsed().andThen(spec).accept(kotlin);
+//        Consumer<K.CompilationUnit> consumer = kotlin.getAfterRecipe().andThen(isFullyParsed()).andThen(spaceConscious(kotlin));
```

Comment:
By commenting in this line and commenting out the next line, the assertion can for local testing be enabled globally.